ios/engine/KMEI/KeymanEngine/Classes/KeymanPackage.swift

```diff
@@ -9,13 +9,21 @@
 import Foundation
 import Zip
 
+// KMPErrors may be passed to UIAlertControllers, so they need localization.
 public enum KMPError : String, Error {
-  case noMetadata = "Could not find kmp.json for package."
-  case invalidPackage = "Invalid Keyman Package."
-  case fileSystem = "Unable to create directory structure in file system."
-  case copyFiles = "Unable to copy keyboard files to file system."
-  case wrongPackageType = "Package provided does not match the type expected by this method"
-  case resourceNotInPackage = "Resource cannot be found in specified package"
+  // The value:  the translation-lookup key.
+  case noMetadata = "kmp-error-no-metadata"
+  case invalidPackage = "kmp-error-invalid"
+  case fileSystem = "kmp-error-file-system"
+  case copyFiles = "kmp-error-file-copying"
```

Comment:
These keys are getting included in the Context on crowdin. Is that intentional?

![image](https://user-images.githubusercontent.com/7358010/88881533-d8d54800-d259-11ea-9280-470ddec0e77a.png)

Reply:
Crowdin shows keys in the context automatically

Reply:
I mean, I'd rather it not use the key as the first part of the Context.  That's kind of on how Crowdin imports the files, though.  Know of a setting that'd fix it?

Reply:
It's not such a big deal in the translation editor. I am okay with it being visible.

![image](https://user-images.githubusercontent.com/4498365/88882201-c1578880-d274-11ea-940e-b2197afd4858.png)

Reply:
ah, I see the Android strings also include the string `name` in the context fields...

Reply:
For comparison, this is a relevant portion of the file that's actually uploaded:

![Screen Shot 2020-07-30 at 12 19 46 PM](https://user-images.githubusercontent.com/25213402/88883621-01137580-d25f-11ea-8475-6c32b178f50d.png)

The file _must_ contain those keys for localization to work.